src/main/kotlin/com/workos/directorysync/models/User.kt

```diff
@@ -33,6 +33,10 @@ open class User
   @JsonProperty("directory_id")
   open val directoryId: String,
 
+  @JvmField
+  @JsonProperty("organization_id")
+  open val organizationId: String?,
```

Comment:
Can we add this field to the Javadoc above?

Reply:
Done